src/main/java/heronarts/lx/effect/BlurEffect.java

```diff
@@ -100,7 +104,7 @@ public void run(double deltaMs, double amount) {
       // Apply exponential decay to the blur
       blurColors[i] = LXColor.multiply(blurColors[i], decayColor, LXColor.BLEND_ALPHA_FULL);
       // Add the new blur buffer frame
-      blurColors[i] = LXColor.add(blurColors[i], this.colors[i], LXColor.BLEND_ALPHA_FULL);
+      blurColors[i] = LXColor.lightest(blurColors[i], this.colors[i], LXColor.BLEND_ALPHA_FULL);
```

Comment:
Yup, annoyed I got this wrong initially and that it's now a "breaking" change (in the sense that behavior will look different for same settings in a saved project).

But I agree this is definitely how it *should* behave. Def shouldn't accumulate more brightness than is actually present in the source material.

Just nuke the `isEnabled()` check and we're good.